lib/metrics/drivers/dog_statsd.rb

```diff
@@ -25,7 +25,7 @@ def emit(instrumenter)
     private
 
     def formatted_tags(tags)
-      tags.map { |k, v| "%s:%s" % [k, v] }
+      tags.map { |k, v| "%s:%s" % [k, v.tr(' ', '_')] }
```

Comment:
that's good; I suggest to call `to_s` as well (in case someone passes in a symbol or a `Numeric` or something else), i.e. `v.to_s.tr(' ', '_')`